vulnerability-analyzer/src/main/java/org/hyades/config/SnykConfig.java

```diff
@@ -19,6 +19,8 @@ public interface SnykConfig {
 
     List<SeveritySource> severitySourcePriority();
 
+    List<String> supportedPackageUrl();
```

Comment:
May I suggest using `supportedPurlTypes` instead? As this is a list the variable name should be plural. And because it is about the type part of the PURL, not the PURL entirely, it should say "types" to avoid any ambiguity.

Reply:
yes I changed this myself. I also thought it is not good enough name 👍